pr-data.csv

```diff
@@ -1668,11 +1668,11 @@ https://github.com/apache/incubator-streampipes,7ba9213012b72ea70edc6848c18ba6d9
 https://github.com/apache/incubator-wayang,b7b6c1ebf65b58f5c03ac2b9fd4c1846ea877d01,wayang-platforms.wayang-graphch,org.apache.wayang.graphchi.operators.GraphChiPageRankOperatorTest.testExecution,ID,,,
 https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.log.snapshot.MetaSimpleSnapshotTest.testSerialize,ID,Opened,https://github.com/apache/iotdb/pull/4380,
 https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.log.snapshot.PullSnapshotTaskTest.testWithRetry,ID,Opened,https://github.com/apache/iotdb/pull/4380,
-https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.query.ClusterPlanExecutorTest.testGetAllStorageGroupNodes,ID,,,
+https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.query.ClusterPlanExecutorTest.testGetAllStorageGroupNodes,ID,Deleted,,https://github.com/apache/iotdb/commit/11c6c331e61a127a3c66906d5dfde75069ec04c6
 https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.query.reader.mult.AssignPathManagedMergeReaderTest.testMultManagerMergeRemoteSeriesReader,ID,Opened,https://github.com/apache/iotdb/pull/4380,
 https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.query.reader.mult.RemoteMultSeriesReaderTest.testAsyncMultSeriesReader,ID,Opened,https://github.com/apache/iotdb/pull/4380,
 https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.query.reader.mult.RemoteMultSeriesReaderTest.testSyncMultSeriesReader,ID,Opened,https://github.com/apache/iotdb/pull/4380,
-https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.server.member.MetaGroupMemberTest.testRemoteAddNode,ID,,,
+https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.server.member.MetaGroupMemberTest.testRemoteAddNode,ID,Deleted,,https://github.com/apache/iotdb/commit/11c6c331e61a127a3c66906d5dfde75069ec04c6
 https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.utils.SerializeUtilTest.testCreateMultiTimeSeriesPlanLog,ID,,,
```

Comment:
Was `SerializeUtilTest` also deleted if they deleted the entire module?

Reply:
Yes it was. Updated the PR.